mozilla-release/mobile/android/base/java/org/mozilla/gecko/preferences/GeckoPreferenceFragment.java

```diff
@@ -94,13 +98,31 @@ public void onCreate(Bundle savedInstanceState) {
         PreferenceScreen screen = getPreferenceScreen();
         setPreferenceScreen(screen);
         /* Cliqz start */
-        // remove Home setting from General settings
         if(res == R.xml.preferences_general || res == R.xml.preferences_general_tablet){
+            // remove Home setting from General settings
             PreferenceScreen homePreferenceScreen = (PreferenceScreen) screen.findPreference
                     (PREF_GENERAL_HOME);
             if(homePreferenceScreen != null) {
                 screen.removePreference(homePreferenceScreen);
             }
```

Comment:
This lines look like a candidate for a function, do not repeat yourself: create a `removePreference` method instead